### ISSUE TITLE
Review in Studio

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-quadratics#151]
In #137, @amanda-phet said:
> @kathy-phet and I are thinking we should do another look at studio for this sim since there have likely been some changes to common code and studio practices since we last worked on this sim.
> 
> If we have changes to make to the featured list, we can do that with overrides. If anything else comes up, we will let you know @pixelzoom.

I'm promoting this to its own issue because @kathy-phet asked me yesterday about the status of this sim.  This is currently the only task that needs to be done before this sim is ready for QA.

**Comment by pixelzoom** [comment]
Oops - duplicate of https://github.com/phetsims/graphing-quadratics/issues/139. Closing.
